### ISSUE TITLE
ng-strict-di fails

### ISSUE THREAD

**Issue by johanssonpeter** [FlaxHaxx/angular-input-types#1]
When I load the minified version and decorate my  app with ng-strict-di. E.g. `<html ng-app="root" ng-strict-di lang="sv-se">`. The library fails to load. Removing the decoration and it works.
```
Error: $injector:strictdi
Explicit annotation required

function($browser,validate is not using explicit annotation and cannot be invoked in strict mode
```

**Comment by FlaxHaxx** [comment]
Thanks for reporting the issue with strictdi. I have fixed it and commited a new version. Please try again with https://raw.githubusercontent.com/FlaxHaxx/angular-input-types/master/dist/angular-input-types.min.js.

I will publish the new version to npm after I have tested it better.

**Comment by johanssonpeter** [comment]
That minified version works for me.

Thanks for the very quick response. Looking forward to the npm version.

**Comment by FlaxHaxx** [comment]
Fixed with commit 1bd013c38a777722b6bfd26612a4b269d7bdf50f
